Source/FolioReaderKit.swift

```diff
@@ -97,29 +97,48 @@ public class FolioReader: NSObject {
     }
     
     /// Check if current theme is Night mode
-    static var nightMode: Bool {
+    public static var nightMode: Bool {
         get { return FolioReader.defaults.boolForKey(kNightMode) }
         set (value) {
             FolioReader.defaults.setBool(value, forKey: kNightMode)
+			FolioReader.defaults.synchronize()
+
+			if let _readerCenter = FolioReader.sharedInstance.readerCenter {
+				UIView.animateWithDuration(0.6, animations: {
+					_readerCenter.currentPage?.webView.js("nightMode(\(nightMode))")
+					_readerCenter.pageIndicatorView?.reloadColors()
+					_readerCenter.configureNavBar()
+					_readerCenter.scrollScrubber?.updateColors()
```

Comment:
Why not use same name here? `updateColors` and `reloadColors`, let's pick one 😄.